imports/utils/location.js

```diff
@@ -0,0 +1,78 @@
+// TODO: Error handling
+
+import { logger } from '/imports/utils/logger';
+import { escapeString } from '/imports/api/utils';
+
+const BASE_URL = 'https://geocode.xyz/';
+const LOCATION_ERROR = 'Unable to get location, please enter address manually';
+
+function makeGeocodeAPICall(location) {
+  // TODO: When a lat/long is passed in this returns 1 main address if found but can also return
+  // multiple "alt" addresses, user should confirm which one because the main address
+  // isn't always the correct one
+  const url = `${BASE_URL}?locate=${encodeURIComponent(location)}&json=1`;
+
+  return new Promise((resolve, reject) => {
+    Meteor.call('surveys.getGeocodedLocation', url, (err, res) => {
+      if (err) {
+        reject(err);
+      } else {
+        resolve(res);
+      }
+    });
+  });
+}
+
+export function getAddressFromLatLong(latLng) {
+  const latLngStr = latLng.join(',');
+  let address = '';
+  makeGeocodeAPICall(latLngStr)
+    .then(result => {
+      if (result.error) {
+        logger.error(escapeString(response.error.description));
+      }
+
+      if (result.standard) {
+        const stnumber = result.standard.stnumber ? result.standard.stnumber : '';
```

Comment:
for those cases you can also write:
`const stnumber = result.standard.stnumber || '';`

Reply:
This code was removed when we changed APIs.